pkgs/by-name/fa/fawltydeps/package.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  writableTmpDirAsHomeHook,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "fawltydeps";
+  version = "0.20.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "tweag";
+    repo = "FawltyDeps";
+    tag = "v${version}";
+    hash = "sha256-RGwCi4SD0khuOZXcR9Leh9WtRautnlJIfuLBnosyUgk=";
+  };
+
+  build-system = with python3Packages; [ poetry-core ];
+
+  dependencies = with python3Packages; [
+    pyyaml
+    importlib-metadata
+    isort
+    pip-requirements-parser
+    pydantic
+  ];
+
+  nativeCheckInputs =
+    [
+      writableTmpDirAsHomeHook
+    ]
+    ++ (with python3Packages; [
+      pytestCheckHook
+      hypothesis
+    ]);
+
+  disabledTestPaths = [
+    # Try to access files.pythonhosted.org
+    "tests/test_install_deps.py"
+    "tests/test_local_env.py"
+    "tests/test_packages.py"
+    "tests/test_resolver.py"
+    "tests/test_settings.py"
+    "tests/test_traverse_project.py"
+    "tests/test_types.py"
+  ];
+
+  meta = {
+    description = "Find undeclared and/or unused 3rd-party dependencies in your Python project";
+    homepage = "https://tweag.github.io/FawltyDeps";
+    license = lib.licenses.mit;
+    mainProgram = "fawltydeps";
+    maintainers = with lib.maintainers; [ aleksana ];
```

Feel free to add me as a maintainer.

```suggestion
    maintainers = with lib.maintainers; [ aleksana jherland ];
```